### ISSUE TITLE
tt100k yolov5s6

### ISSUE THREAD

**Issue by hbo-lambda** [ultralytics/yolov5#11655]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and [discussions](https://github.com/ultralytics/yolov5/discussions) and found no similar questions.

### Question

environment:
nvidia: v100 32G
ubuntu: 20.04
python: 3.7
cuda: 11.2
torch: 1.10.1+cu111

I am training road sign recognition using tt100k data set, pre-training model is yolov5s6.
The image size in tt100k data set is 2048*2048, so initially I wanted to use yolov5s6 for training
I filtered the number of tags greater than 100 in tt100k.

When I use the following command, the first epoch will break while evaluating the test set and the physical server will restart:
```
python train.py --img 1280 --epoch 300 --data roadsign.yaml --weights yolov5s6.pt
```
I tried the following command again, the same phenomenon：
```
python train.py --img 1280 --epochs 300 --data roadsign.yaml --weights yolov5s.pt
```
When I changed the image size, the training process was successful and the physical server did not restart
```
python train.py --img 640 --epochs 300 --data roadsign.yaml --weights yolov5s.pt
```
I just can't figure out what went wrong

**Comment by glenn-jocher** [comment]
Hello @hbo-lambda, 

It seems like the issue is related to the image size of the tt100k dataset. The size of 2048x2048 is quite large and could be exceeding your GPU's memory capacity, causing the physical server to restart. By reducing the image size with the `--img` parameter, the training process was successful. This is a common issue and can be resolved by choosing appropriate image size according to the GPU's memory capacity. 

Let us know if you have any other issues or questions. 

Thank you.

**Comment by hbo-lambda** [comment]
I'm just curious why.  below is a screenshot of me training with 960 dimensions, gpu 15327MiB / 32768MiB. server reboot....
![Snip20230609_1](https://github.com/ultralytics/yolov5/assets/7626261/186379e4-74de-40d4-b09c-1b28acb9fa6c)

**Comment by glenn-jocher** [comment]
@hbo-lambda hello,

It appears that the image size of 960 dimensions is causing your GPU to exceed its memory capacity, which leads to a physical server reboot. This issue can occur frequently during training when the image size is too large according to your GPU memory. Higher image size leads to more computation and memory requirements, and too much memory usage will ultimately cause the GPU to crash. To avoid this issue, you can try reducing the image size, or you can consider using a more powerful GPU. 

Hope this helps! Let us know if you have any other questions.
